app/components/submission/ManuscriptMetadata/SubjectAreaDropdown.js

```diff
@@ -0,0 +1,165 @@
+  },
+  {
+    value: 'developmental-biology-stem-cells',
+    label: 'Developmental Biology and Stem Cells',
+  },
+  { value: 'ecology', label: 'Ecology' },
+  {
+    value: 'epidemiology-global-health',
+    label: 'Epidemiology and Global Health',
+  },
+  { value: 'evolutionary-biology', label: 'Evolutionary Biology' },
+  { value: 'genetics-genomics', label: 'Genetics and Genomics' },
+  {
+    value: 'human-biology-medicine',
+    label: 'Human Biology and Medicine',
+  },
+  {
+    value: 'immunology-inflammation',
+    label: 'Immunology and Inflammation',
+  },
+  {
+    value: 'microbiology-infectious-disease',
+    label: 'Microbiology and Infectious Disease',
+  },
+  { value: 'neuroscience', label: 'Neuroscience' },
+  { value: 'physics-living-systems', label: 'Physics of Living Systems' },
+  { value: 'plant-biology', label: 'Plant Biology' },
+  {
+    value: 'structural-biology-molecular-biophysics',
+    label: 'Structural Biology and Molecular Biophysics',
+  },
+]
+
+const findOptionByValue = (value, options) =>
+  options.find(option => option.value === value)
+
+export class ThemelessSubjectAreaDropdown extends React.Component {
+  constructor(props) {
+    super(props)
+
+    const savedOptions = this.props.savedValues.map(value =>
+      findOptionByValue(value, eLifeOptions),
+    )
+
+    this.state = {
+      selectedOptions: savedOptions,
+      hasReachedMultiselectLimit: false,
+    }
+
+    this.handleChange = this.handleChange.bind(this)
+    this.handleBlur = this.handleBlur.bind(this)
+
+    this.customReactSelectStyles = {
+      multiValue: (base, state) => ({
+        ...base,
+        backgroundColor: this.props.theme.colorPrimary,
+      }),
+    }
+  }
+
+  // https://github.com/JedWatson/react-select/blob/v2/src/Select.js
+  // react-select exposes onChange: (ValueType, ActionMeta) => void
+  handleChange = selectedOptions => {
+    if (selectedOptions.length <= 2) {
+      this.setState({ selectedOptions, hasReachedMultiselectLimit: false })
+      if (this.props.onChange) this.props.onChange(selectedOptions)
+    }
+    if (selectedOptions.length >= 2) {
+      this.setState({ hasReachedMultiselectLimit: true })
+    }
+  }
+
+  handleBlur = e => {
+    if (this.props.onBlur) this.props.onBlur(e)
+  }
+
+  render() {
+    const { selectedOptions, hasReachedMultiselectLimit } = this.state
+    const { label, name } = this.props
+
+    const selectChildProps = {
+      components: { ClearIndicator: null },
+      inputId: 'subject-area-select',
+      isMulti: true,
+      name,
+      onBlur: this.handleBlur,
+      onChange: this.handleChange,
+      options: eLifeOptions,
+      styles: this.customReactSelectStyles,
+      value: selectedOptions,
+      filterOption: (option, rawInput) => {
+        const input = rawInput.replace(/^\s+|\s+$/g, '').toLowerCase()
+        const candidate = `${option.label} ${option.value}`
+          .replace(/^\s+|\s+$/g, '')
```

Comment:
regexp could maybe use a comment

Reply:
"strip whitespace from beginning and end of string"

Reply:
aka "trim"!

Reply:
https://www.w3schools.com/jsref/jsref_trim_string.asp n'est ce pas?

Reply:
Good point! I copied this directly from [the default filter function used within `react-select`](https://github.com/JedWatson/react-select/blob/v2/src/filters.js#L13). I assume he's written it out explicitly with a regex, rather than using the `.trim` method, because of this line in the w3schools article that you linked:
> For browsers that do not support the trim() method, you can remove whitespaces from both sides of a string with a regular expression

But in his implementation, he does name this regex function - 'trimString', which is definitely more readable :+1:

Reply:
Interestingly, MDN seems to suggest that it is supported across all browsers
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim

Reply:
Ok, realised I can actually configure the existing createFilter function & don't need to define my own. Makes far more sense, since I was copying all this code from there anyway :sweat_smile: